lib/tapioca/cli.rb

```diff
@@ -311,5 +321,63 @@ def self.exit_on_failure?
         true
       end
     end
+
+    private
+
+    def print_init_next_steps
+      say(<<~OUTPUT)
+        #{set_color("This project is now set up for use with Sorbet and Tapioca", :bold)}
+
+        The sorbet/ folder should exist and look something like this:
+
+        ├── config             # Default options to be passed to Sorbet on every run
+        └── rbi/
+          ├── annotations/     # Type definitions pulled from the rbi-central repository
+          ├── gems/            # Autogenerated type definitions for your gems
+          └── todo.rbi         # Constants which were still missing after RBI generation
+        └── tapioca/
+          ├── config.yml       # Default options to be passed to Tapioca
+          └── require.rb       # A file where you can make requires from gems that might be needed for gem RBI generation
+
+        Please check this folder into version control.
+
+        #{set_color("🤔 What's next", :bold)}
+
+        1. Many Ruby applications use metaprogramming DSLs to dynamically generate constants and methods.
+          To generate type definitions for any DSLs in your application, run:
+
+          #{set_color("bin/tapioca dsl", :cyan)}
+
+        2. Check whether the constants in the #{set_color("sorbet/rbi/todo.rbi", :cyan)} file actually exist in your project.
+          It is possible that some of these constants are typos, and leaving them in todo.rbi will
+          hide errors in your application. Ideally, you should be able to remove all definitions
+          from this file and stop using todo.rbi entirely.
+
+        3. Typecheck your project:
+
+          #{set_color("bundle exec srb tc", :cyan)}
+
+          There should not be any typechecking errors.
+
+        4. Upgrade a file marked "#{set_color("# typed: false", :cyan)}" to "#{set_color("# typed: true", :cyan)}".
+          Then, run: #{set_color("bundle exec srb tc", :cyan)} and try to fix any errors.
+
+          If you are using Spoom, you may run:
```

Comment:
@Morriar I mentioned spoom here as an alternate way of bumping files because I'm sure not everyone will know what Spoom is. Let me know if you think this is enough.

Reply:
Tapioca depends on Spoom so we may just give the command:

```md
You can use Spoom to bump files for you:

#{set_color("spoom bump --from false --to true", :cyan)}

To learn more about Spoom, visit: #{set_color("https://github.com/Shopify/spoom", :cyan)}
```